test/web-auth/captcha.test.js

```diff
@@ -172,52 +182,75 @@ describe('captcha rendering', function () {
     });
 
     it('should set the image tag', function () {
-      expect(element.innerHTML).to.equal(`custom template: ${challenges[0].image}`);
+      expect(element.innerHTML).to.equal(
+        `custom template: ${challenges[0].image}`
+      );
     });
   });
 
   const RECAPTCHA_V2_PROVIDER = 'recaptcha_v2';
   const RECAPTCHA_ENTERPRISE_PROVIDER = 'recaptcha_enterprise';
   const HCAPTCHA_PROVIDER = 'hcaptcha';
   const FRIENDLY_CAPTCHA_PROVIDER = 'friendly_captcha';
-
-
-  [RECAPTCHA_V2_PROVIDER,RECAPTCHA_ENTERPRISE_PROVIDER, HCAPTCHA_PROVIDER, FRIENDLY_CAPTCHA_PROVIDER].forEach(provider => {
+  const ARKOSE_PROVIDER = 'arkose';
+
+  [
+    RECAPTCHA_V2_PROVIDER,
+    RECAPTCHA_ENTERPRISE_PROVIDER,
+    HCAPTCHA_PROVIDER,
+    FRIENDLY_CAPTCHA_PROVIDER
+  ].forEach(provider => {
```

Comment:
Should we add ARKOSE_PROVIDER in this array?

Reply:
See https://github.com/auth0/auth0.js/pull/1341#discussion_r1346953330

Reply:
Should we get rid of the const added above it for arkose?

Reply:
See https://github.com/auth0/auth0.js/pull/1341#discussion_r1346960213